src/common/models/bmcapimodels/servermodels/testutil_servergenerator.go

```diff
@@ -0,0 +1,205 @@
+package servermodels
+
+import (
+	"math/rand"
+	"time"
+
+	bmcapisdk "github.com/phoenixnap/go-sdk-bmc/bmcapi"
+
+	ranchersdk "github.com/phoenixnap/go-sdk-bmc/ranchersolutionapi"
+	testutil "phoenixnap.com/pnapctl/testsupport/testutil"
+)
+
+func GenerateServerListSdk(n int) []bmcapisdk.Server {
+	var serverlist []bmcapisdk.Server
+	for i := 0; i < n; i++ {
+		serverlist = append(serverlist, GenerateServerSdk())
+	}
+	return serverlist
+}
+
+func GenerateServerSdk() bmcapisdk.Server {
+	provisionedOn := time.Now()
+	return bmcapisdk.Server{
+		Id:                 testutil.RandSeq(10),
+		Status:             testutil.RandSeq(10),
+		Hostname:           testutil.RandSeq(10),
+		Description:        testutil.RandSeqPointer(10),
+		Os:                 testutil.RandSeq(10),
+		Type:               testutil.RandSeq(10),
+		Location:           testutil.RandSeq(10),
+		Cpu:                testutil.RandSeq(10),
+		CpuCount:           int32(rand.Int()),
+		CoresPerCpu:        int32(rand.Int()),
+		CpuFrequency:       rand.Float32(),
+		Ram:                testutil.RandSeq(10),
+		Storage:            testutil.RandSeq(10),
+		PrivateIpAddresses: []string{},
+		PublicIpAddresses:  []string{},
+		ReservationId:      testutil.RandSeqPointer(10),
+		PricingModel:       testutil.RandSeq(10),
+		Password:           testutil.RandSeqPointer(10),
+		NetworkType:        testutil.RandSeqPointer(10),
+		ClusterId:          testutil.RandSeqPointer(10),
+		Tags:               nil,
+		ProvisionedOn:      &provisionedOn,
+		OsConfiguration:    nil,
+	}
+}
+
+func GenerateServerCreateCli() ServerCreate {
+	return ServerCreate{
+		Hostname:              testutil.RandSeq(10),
+		Description:           testutil.RandSeqPointer(10),
+		Os:                    testutil.RandSeq(10),
+		Type:                  testutil.RandSeq(10),
+		Location:              testutil.RandSeq(10),
+		InstallDefaultSshKeys: nil,
+		SshKeys:               nil,
+		SshKeyIds:             nil,
+		ReservationId:         testutil.RandSeqPointer(10),
+		PricingModel:          testutil.RandSeqPointer(10),
+		NetworkType:           testutil.RandSeqPointer(10),
+		OsConfiguration:       nil,
+		Tags:                  nil,
+		NetworkConfiguration:  nil,
+	}
+}
+
+func GenerateClusterListSdk(n int) []ranchersdk.Cluster {
+	var clusterlist []ranchersdk.Cluster
+	for i := 0; i < n; i++ {
+		clusterlist = append(clusterlist, GenerateClusterSdk())
+	}
+	return clusterlist
+}
+
+func GenerateClusterSdk() ranchersdk.Cluster {
```

Comment:
This is already available in the `testutil_ranchergenerator.go`

Reply:
This is a temporary file, copy and pasted from Brian's branch - because I need his generator functions. Once his PR gets merged I'll delete this file and merge.